ui/src/containers/AppDetailsContainer.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import AppsTable from '../components/AppsTable';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { Checkbox } from '@patternfly/react-core';
+import { getApps, reverseAppDetailsTableSort } from '../actions/actions-ui';
+
+export class AppDetailsContainer extends React.Component {
+  componentWillMount () {}
+
+  handleChange = (e) => {
+    console.log('checkbox clicked');
+  };
+
+  onSort = (_event, index) => {
+    this.props.reverseAppDetailsTableSort(index);
+  };
+
+  getTable = () => {
+    const renderedRows = [];
```

Comment:
@damienomurchu I'm doing this transferring in the reducer. I don't think it's a problem to do it here but I think we should try to be consistent. If we do this here for this one we should also do that for the other table.

Reply:
@laurafitzgerald considered that, but the logic is not just re-ordering the data, but transforming it (ie coverting a `true`/ `false` in the state to a checkbox component when rendered. I think the best way to do this is in the component, as otherwise we end up with the reducer needing to store `<Checkbox />` components in the stores state data. For that reason I think that logic belongs best here.

Reply:
Ok. In that case should we consider moving it for the other table and only passing back the data there too?

Reply:
I'm not sure. Let me know if you want to discuss offline. Just going to close this as I don't see it related to this PR, but rather the implementation for the other table.